public/externals.json

```diff
@@ -12,17 +12,17 @@
       {
         "global": "React",
         "import": "react",
-        "path": "https://unpkg.com/react@16.4.2/umd/react.development.js"
+        "path": "https://unpkg.com/react@16.6.1/umd/react.development.js"
```

Comment:
Newest version is [`16.6.3`](https://github.com/facebook/react/blob/master/CHANGELOG.md#1663-november-12-2018) ⬆️